build.gradle

```
@@ -0,0 +1,57 @@
+apply plugin: 'java-library'
+
+sourceSets {
+   main {
+      java {
+         srcDir 'src'
+      }
+   }
+	
+   test {
+      java {
+         srcDir 'test'
+      }
+   }
+}
+
+sourceCompatibility = 1.8
```

Será que não deveriamos migrar o CST para Java 10 ou superior?
```suggestion
sourceCompatibility = 1.11
```

---

OK, thanks, very good suggestion! :)

---

Ended up keeping the sourceCompatibility at 1.8, in respect to old projects like WS3D.